docs/deploy-from-brew.md

```diff
@@ -0,0 +1,196 @@
+kind: ImageContentSourcePolicy
+metadata:
+  name: brew-registry
+spec:
+  repositoryDigestMirrors:
+  - mirrors:
+    - brew.registry.redhat.io
+    source: registry.redhat.io
+  - mirrors:
+    - brew.registry.redhat.io
+    source: registry.stage.redhat.io
+  - mirrors:
+    - brew.registry.redhat.io
+    source: registry-proxy.engineering.redhat.com
+EOF
+```
+
+## Deploying ACM from Brew
+
+### The Easy Way (RECOMMNDED)
+
+Run `./start-brew.sh` for a simple deploy experience.  
+
+This does the same thing as the manual steps documented below, only documented and with additional error checking!
+
+You can optionally set these configuration parameters (defaults shown at time of writing, may not remain up to date):
+```
+# Define your CatalogSource (CatalogSource will reference 'image: ${CATALOGSOURCE_REPO}/${CATALOGSOURCE_IMAGE}:${CATALOGSOURCE_TAG}')
+CATALOGSOURCE_REPO      -   The repo that holds the CatalogSource image, default: "brew.registry.redhat.io/rh-osbs"
+CATALOGSOURCE_IMAGE     -   The name of the CatalogSource image found in CATALOGSOURCE_REPO, default: "iib-pub-pending"
+CATALOGSOURCE_TAG       -   The tag on the Catalog image to use, default: "v4.7"
+
+# MultiCluster Operator Subscription Channel
+SUBSCRIPTION_CHANNEL    -   The subscription channel used for the MCH operator, default: "release-2.3"
+
+# Deploy Config
+TARGET_NAMESPACE        -   The namespace in which to deploy the CatalogSoruce, Operator, and MultiClusterHub, default: "open-cluster-management"
+
+# Naming
+CATALOGSOURCE_RESOURCE_NAME -   The name of the CatalogSource resource, default: "start-brew-iib"
+SUBSCRIPTION_NAME           -   The name of the Subscription resource, default: "start-brew-sub"
+```
+
+### The Hard Way (NOT recomended)
+
+#### Step 0: Disable Old CatalogSources
+
+```
+oc patch OperatorHub cluster --type json -p '[{"op": "add", "path": "/spec/disableAllDefaultSources","value": true}]'
+```
+
+#### Step 1: Create the open-cluster-management Namespace
+
+```
+oc create ns open-cluster-management
+```
+
+#### Step 2: Add the Custom CatalogSource
+
+```
+oc apply -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: CatalogSource
+metadata:
+  name: start-brew-iib
+  namespace: ${TARGET_NAMESPACE}
+spec:
+  sourceType: grpc
+  image: ${CATALOGSOURCE_REPO}/${CATALOGSOURCE_IMAGE}:${CATALOGSOURCE_TAG}
+  displayName: Start Brew iib ${CATALOGSOURCE_TAG}
+  publisher: grpc
+EOF
+```
+
+#### Step 4: Create an OperatorGroup
+
+```
+oc apply -f - <<EOF
+apiVersion: operators.coreos.com/v1
+kind: OperatorGroup
+metadata:
+  name: default
+  namespace: ${TARGET_NAMESPACE}
+spec:
+  targetNamespaces:
+  - ${TARGET_NAMESPACE}
+EOF
+```
+
+#### Step 5: Create a Subscription
+
+```
+oc apply -f - <<EOF
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: ${SUBSCRIPTION_NAME}
+  namespace: ${TARGET_NAMESPACE}
+spec:
+  channel: ${SUBSCRIPTION_CHANNEL}
+  installPlanApproval: Automatic
```

Comment:
This needs to be parameterized because the upgrade process needs it to be set to manual.

Reply:
I think, for this initial push, the brew deploy shouldn't be our upgrade-supported deploy method and we can patch that in as additional functionality in a separate story (it wasn't in scope for this story, but not hard to add once we know the dynamics of upgrade with brew).